src/Generator/Generators/HtmlOmdGenerator.cs

```diff
@@ -261,7 +262,7 @@ private string GetIcon(ISymbol type, string content)
             }
             if (icon == "")
                 return content;
-            return $"<li class='{icon}'>{content}</li>";
+            return $"<li class='{icon} code'>{content}</li>";

```

Comment:
I think it's a good idea to create a style users can easily override, but not sure this should be the style name, or that it should default to a mono-space font.